src/drone_controller/input_layer/request_handler.py

```diff
@@ -0,0 +1,41 @@
+"""
+This module handles all requests. This will cover REST calls as well as native python calls.
+"""
+from src.drone_controller.calculation_layer.thrust_calculator import ThrustCalculator
+from src.drone_controller.exception.exceptions import UserInputError
+from src.drone_controller.input_layer.drone_state import DroneState
+from src.drone_controller.input_layer.drone_state_mapper import DroneStateMapper
+
+
+class RequestHandler:
```

Comment:
With this class setup, every step you call `RequestHandler.keyboard_input(x, y)` and create a new Instance of RequestHandler, DroneStateMapper and ThrustCalculator.

Can't you init all classes once and than siply call their methods?